sdk/tests/deqp/framework/common/tcuTestCase.js

```diff
@@ -271,7 +272,6 @@ goog.scope(function() {
 
         // If no more children, get the next brother
         if (test == null && this.parentTest != null) {
-            this.currentTestNdx = 0;
```

Comment:
It will print wrong skip test information at the end of a test html. Have you seen them before? You can try texturefunction* tests to see them. These information is generated when go through brother tests of currentTest's parent (though these tests will not be executed because out of range).